pkg/apis/authentication/v1/defaults.go

```diff
@@ -17,9 +17,22 @@ limitations under the License.
 package v1
 
 import (
+	authenticationv1 "k8s.io/api/authentication/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
 func addDefaultingFuncs(scheme *runtime.Scheme) error {
 	return RegisterDefaults(scheme)
 }
+
+var kubernetesServiceAud = []string{"https://kubernetes.default.svc"}
+
+func SetDefaults_TokenRequestSpec(obj *authenticationv1.TokenRequestSpec) {
+	if len(obj.Audiences) == 0 {
+		obj.Audiences = kubernetesServiceAud
+	}
+	if obj.ExpirationSeconds == nil {
+		hour := int64(60 * 60)
+		obj.ExpirationSeconds = &hour
```

Comment:
rationale behind this default? is this anticipated to be a reasonably usable lifetime, or intended to motivate people to specify a value to avoid getting this default?

Reply:
30 or 60 minutes is pretty standard. I anticipate that this will be a reasonably usable lifetime. It's more usable than 0 minutes which is the current default of the field.

Reply:
1 hour seems reasonable